weatherApp/ViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ViewController.swift
+//  weatherApp
+//
+//  Created by Luyanda Sikithi on 2022/11/02.
+//
+
+import UIKit
+import Resolver
+class ViewController: UIViewController {
+
+    @IBOutlet weak var tableView: UITableView!
+    
+    @Injected var viewModel: WeatherViewModel
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupTableView()
+        getWeather()
+    }
+    
+    private func setupTableView() {
+        tableView.dataSource = self
+        tableView.delegate = self
+        
+        tableView.register(UINib(nibName: "CitiesTableViewCell", bundle: nil), forCellReuseIdentifier: "CitiesTableViewCell")
+    }
+    
+    private func getWeather() {
+        viewModel.getWeather()
+    }
+}
+
+extension ViewController: UITableViewDataSource, UITableViewDelegate {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 4
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "CitiesTableViewCell", for: indexPath) as? CitiesTableViewCell else { return UITableViewCell() }
+        if indexPath.row == 0 {
+            cell.cityLbl.text = "Pretoria"
+            cell.countryLbl.text = "South Africa"
+        }else if indexPath.row == 1 {
+            cell.cityLbl.text = "London"
+            cell.countryLbl.text = "United Kindom"
+        }else if indexPath.row == 2 {
+            cell.cityLbl.text = "Barcelona"
+            cell.countryLbl.text = "Europe"
+        }else if indexPath.row == 3 {
+            cell.cityLbl.text = "New York"
```

Comment:
Could we not refactor all of this into an array? And then have the tableView build off that array?
So something like:

countries: [Country] = [Country("South-Africa", "Pretoria", etc...]
Then we get the data based off the row index.